dev/Dockerfile.base

```diff
@@ -32,6 +32,7 @@ RUN set -ex; \
 RUN set -ex; \
     touch /var/lib/rpm/* \
     && dnf -y install \
+        gettext \
```

Comment:
I think we avoided doing this because we didn't want to install an unnecessary dependency in the images that get published to console dot (correct me if I'm wrong @awcrosby)

Reply:
oh, this is the dev image. Ignore me